packages/experimental/Tabs/src/TabsItem.types.ts

```diff
@@ -146,7 +146,7 @@ export interface TabsItemInfo {
 }
 
 export interface TabsItemSlotProps {
-  root: React.PropsWithRef<IViewProps>;
+  root: React.PropsWithRef<PressableProps>;
```

Comment:
Technically we lose the adapter props here. Will have to think about that. Could probably easily make an "IPressableProps" if I _really_ wanted to.

Reply:
Yeah in practice as long as the props we expose for the component stay constant I think we're ok... Even if props that don't match the type are passed in, they're still on the object so they should get passed through, they just will make TS angry if someone tries to modify the root slot directly and add props that don't exist on PressableProps

Reply:
So really it'll be more of a problem for us than for clients.

Reply:
I think type safety is still a but of an open problem in FURN. Adapters is a step towards it but if we enable platforms specific type safety checking via rex-kit, there's still issues. I'll try to spend a day thinking about how best to not regress the experience for ourselves.